traits/tests/test_regression.py

```diff
@@ -189,6 +190,22 @@ def test_delegation_refleak(self):
         # All the counts should be the same.
         self.assertEqual(counts[warmup:-1], counts[warmup+1:])
 
+    def test_hastraits_pickle_deepcopy(self):
```

Comment:
It looks as though this is a single test that tests both the pickling and the deepcopy behaviour in one. I think it should be split into two separate tests.

Reply:
I agree.